assets/catalog/parsed_plugin_list.js

```diff
@@ -527,6 +527,36 @@ myfcn(
     "type": "Collector",
     "description": "Collects Linux avaliable entropy from /proc/sys/kernel/random/entropy_avail.",
     "url": "https://github.com/janczer/snap-plugin-collector-entropy"
+  },
+  {
+    "name": "Slack",
+    "type": "Publisher",
+    "description": "No description available.",
```

Comment:
I've already asked @marcin-krolik in PR https://github.com/intelsdi-x/snap/pull/1625 to fill repo description.